sdk/keyvault/azure-security-keyvault-secrets/test/ut/azure_security_keyvault_secrets_test.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "gtest/gtest.h"
+
+#if defined(BUILD_CURL_HTTP_TRANSPORT_ADAPTER)
+#include <curl/curl.h>
+#endif
+
```

Comment:
Remove this file and use gtest-main ?

Reply:
it was left over from the great merge of yesterday